docs/conf/ssl.md

```diff
@@ -2,9 +2,134 @@
 
 SSL handling can be customized (or disabled) when creating a backend and is backend-specific.
 
-Depending on the underlying backend's client, you can customize SSL settings as follows:
+Depending on the underlying backend's client, you can customize SSL settings.
 
-* `HttpUrlConnectionBackend`: when creating the backend, specify the `customizeConnection: HttpURLConnection => Unit` parameter, and set the hostname verifier & SSL socket factory as required
-* akka-http: when creating the backend, specify the `customHttpsContext: Option[HttpsConnectionContext]` parameter. See [akka-http docs](http://doc.akka.io/docs/akka-http/current/scala/http/server-side/server-https-support.html)
-* async-http-client: create a custom client and use the `setSSLContext` method
-* OkHttp: create a custom client modifying the SSL settings as described [on the wiki](https://github.com/square/okhttp/wiki/HTTPS)
+## SSL Context
+
+Common requirement for handling SSL is creating `SSLContext`. It's required by several backends.
+
+### One way SSL
+
+Example assumes that you have your client key store in `.p12` format. If you have your credentials in `.pem` format covert them using:
+
+`openssl pkcs12 -export -inkey your_key.pem -in your_cert.pem -out your_cert.p12`
+
+Sample code might look like this:
+```scala
```

Comment:
can we enable mdoc here? (using scala mdoc:silent)

that would verify that the code compiles (at least :) ) see the readme how to run docs generation localy

Reply:
I can use mdoc:compile-only here but variables are out of scope for mdoc in further code samples so I haven't used it at first

Reply:
According to the docs: https://scalameta.org/mdoc/docs/modifiers.html, `mdoc:silent` accumulates the definitions. Would be good to enable it as much as possible, it really helps in keeping the docs up-to-date :)